floris/simulation/solver.py

```diff
@@ -76,11 +76,17 @@ def sequential_solver(
     v_wake = np.zeros_like(flow_field.v_initial_sorted)
     w_wake = np.zeros_like(flow_field.w_initial_sorted)
 
-    turbine_turbulence_intensity = (
-        flow_field.turbulence_intensity
-        * np.ones((flow_field.n_findex, farm.n_turbines, 1, 1))
-    )
-    ambient_turbulence_intensity = flow_field.turbulence_intensity
+    # Set up turbulence arrays
+    turbine_turbulence_intensity = flow_field.turbulence_intensities[
+        :, np.newaxis, np.newaxis, np.newaxis
```

Comment:
Use `None` instead of `np.newaxis` for consistency.
Also, I see this same array expansion happens in `FlowField`. What's the relationship between this local variable and the attribute on `FlowField`?

Reply:
Done on the conversion!  On the other point, I'm not positive actually, here I was just following the pre-exisiting patterns for which variables were local and which were on FlowField